src/Bumblebee.IntegrationTests/Setup/ThreadedSessionTests.cs

```diff
@@ -50,7 +54,11 @@ public void when_loading_with_generic_driver_should_return_session_with_correct_
 		[Test]
 		public void when_loading_with_driver_and_custom_settings_should_return_session_with_custom_settings()
 		{
```

Comment:
To remain consistent with your other changes, in the ThreadedSessionTests.cs,  I will make the When pascal case too.